inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -505,11 +492,17 @@ std::shared_ptr<ngraph::Node> V10Parser::createNode(const std::vector<ngraph::Ou
     if (!ngraphNode && opsets.count(params.version)) {
         auto opset = opsets.at(params.version);
 
-        if (!opset.contains_type(params.type)) {
+        // ROIPooling was missing in opset1 and was added in opset2
```

Comment:
Am I right that MO generates IR where `ROIPooling` exists in the `opset1`?

@lazarevevgeny I think we should fix it on the MO side if MO generates `opset1` for `ROIPooling`

Reply:
In the ngraph reader IE Functional tests we have IRs with ROIPooling in opset1.
There is also a comment in opset2_tbl file that says that this op should be available in opset1 but was not by mistake, hence my comment here and this fix.

Reply:
I think that we can change the version in the test, instead of this WA. But also need to check that MO generates the right version.

Reply:
I checked that MO from 2020.3 generated ROIPooling with opset1. Starting from 2020.4 MO generates ROIPooling with opset2.

Reply:
Hmm, ok, it means that we need this code, because 2020.3 is LTS release.